internal/resources/grafana/resource_alerting_rule_group.go

```diff
@@ -372,6 +375,7 @@ func packRuleData(queries []*gapi.AlertQuery) (interface{}, error) {
 		if queries[i] == nil {
 			continue
 		}
+		log.Printf("[DEBUG] PACKING!!!\n")
```

Comment:
Nit: We should probably drop this one before submit :smile:

the others are :+1:

Reply:
Hahaha.... oops.